libs/blocks/featured-article/featured-article.js

```diff
@@ -0,0 +1,28 @@
+import {
+  buildArticleCard,
+  getBlogArticle,
+  loadTaxonomy,
+} from '../article-feed/article-helpers.js';
+
+async function decorate(block, article) {
+  const card = buildArticleCard(article, 'featured-article');
+  block.insertAdjacentElement('afterbegin', card);
+  const tagHeader = document.querySelector('.tag-header-container > div');
+  /* c8 ignore next */
+  if (tagHeader) tagHeader.append(block);
+}
+
+export default async function init(block) {
+  const a = block.querySelector('a');
+  /* c8 ignore next */
+  if (!a && !a.href) return;
+  block.innerHTML = '';
+  loadTaxonomy();
```

Comment:
we should probably await this since it's async and I'm guessing you need it for later stuff in your code
`loadTaxonomy();`